libs/hdf-converters/src/twistlock-mapper.ts

```diff
@@ -118,16 +120,44 @@ export class TwistlockMapper extends BaseConverter {
       }
     ],
     passthrough: {
-      raw: {
-        transformer: (
-          data: Record<string, unknown>
-        ): Record<string, unknown> => {
-          return data;
-        }
+      transformer: (data: Record<string, unknown>): Record<string, unknown> => {
+        return {
+          auxiliary_data: [
+            {
+              name: 'Twistlock',
+              data: {
+                results: [
```

Comment:
Please see my response in the earlier review (idk if it'll properly show up so i'mma copy paste the body here too).

----

Oh I see what you mean. The thing you need to keep in mind is that doing the parameter overwriting like we're doing here with the assignment occurring to an internal structure of the object is that the changes will propagate due to js being a pass reference by value type of language. This is a really solid stackoverflow answer covering this with a nice example: [stackoverflow.com/a/3638034/645647](https://stackoverflow.com/a/3638034/645647). Ways to work around this are to make 'auxData' a full and proper clone or to manually specify the missing data in auxiliary_data which could probably be eased through judicious use of the spread operator like so:

```js
const a = {a: 5, b: [{c: 2, d: 4}]};
console.log({...a, b: [_.omit(a.b[0], ['c'])]});
```

which results in

```js
{
  a: 5,
  b: [{
  d: 4
}]
}
```